model/config.go

```diff
@@ -752,6 +753,18 @@ func (o *Config) SetDefaults() {
 		*o.EmailSettings.EmailBatchingInterval = EMAIL_BATCHING_INTERVAL
 	}
 
+	if o.EmailSettings.EnableSMTPAuth == nil {
+		o.EmailSettings.EnableSMTPAuth = new(bool)
+		if o.EmailSettings.ConnectionSecurity == CONN_SECURITY_NONE {
+			*o.EmailSettings.EnableSMTPAuth = false
+		} else {
+			*o.EmailSettings.EnableSMTPAuth = true
+		}
+		if o.EmailSettings.ConnectionSecurity == CONN_SECURITY_PLAIN {
```

Comment:
Can you move this block so that it's still ran when `EnableSMTPAuth` is not nil? I don't know if it's super necessary, but it would be nice to know that `ConnectionSecurity` can never equal `CONN_SECURITY_PLAIN` elsewhere